internetarchive/openlibrary
# Refactor list view_body template file to move inline JavaScript

Addresses a subtask that is part of issue #2468


This PR moves inline JS out of the following template file: `openlibrary/templates/type/list/view_body.html`

### Technical

Inline CSS has been moved to a new CSS file, `static/css/components/list--view-body.less` and inline JS has been moved to a new JS file, `openlibrary/plugins/openlibrary/js/lists/ListViewBody.js`.

### Testing

There should be no visible change to an individual Lists page after this update. To test that no functionality is broken, please:
* Visit a List page with at least two items
* Click the 'Remove this item?' link for a list item
  * Verify that dialog text is unchanged from current prod version
  * Verify that "No, cancel" button successfully cancels the action and that "Yes, I'm sure" button successfully removes the item from the list
* Click the 'Remove this item?' link for the only remaining list item
  * Verify that dialog text is unchanged from current prod version
  * Verify that "No, cancel" button successfully cancels the action and that "Yes, I'm sure" button successfully deletes the list

### Screenshot

<img width="1452" alt="Screen Shot 2023-04-05 at 12 28 49 AM" src="https://user-images.githubusercontent.com/13765801/230011675-252ea676-57d2-4633-8cbf-b26fb062e3f1.png">

<img width="1452" alt="Screen Shot 2023-04-05 at 12 28 57 AM" src="https://user-images.githubusercontent.com/13765801/230011726-6ee40d1e-efcf-4001-80af-79122afaf13b.png">


### Stakeholders

@jdlrobson & @DebbieSan
